sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -374,6 +374,16 @@ querySpecification
        windows?)
     ;
 
+hint
+    : '/*+' hintStatement '*/'
+    ;
+
+hintStatement
+    : hintName=identifier
+    | hintName=identifier '(' parameters+=identifier parameters+=identifier ')'
```

Comment:
The purpose of this is to allow `SqlBase.g4` accept more general rule for the future.
So, possibly, it's for reducing the change in `SqlBase.g4` in order to add new hint rules.